server/scheduler/worker.go

```diff
@@ -56,9 +59,10 @@ func (w *Worker) String() string {
 // that the worker can add itself to when it is idle. It also creates a slice for storing totalProcessed requests.
 func newWorker(workerId int, workerPriority proto.Priority) *Worker {
 	return &Worker{
-		ID:       workerId,
-		Priority: workerPriority,
-		Quit:     make(chan struct{}),
+		ID:         workerId,
+		Priority:   workerPriority,
+		HSMTimeout: config.DefaultPKCS11Timeout * time.Second,
```

Comment:
nit: can we avoid dependency on `config` package by passing it as param from the top-level function in `server.go`?

Reply:
Sure will clean it up in a follow up PR.